[IMP] Lunch: Update manage user accounts (#16076)
odoo/documentation: content/applications/hr/lunch/user-accounts.rst

Reviewer: ```suggestion
The _Control Accounts_ dashboard is a view-only list of transactions within the **Lunch** app. To modify transaction records... .
```

Reviewer: ```suggestion
  Product-related records cannot be modified.
```

Author: Nice change!

A blank :guilabel:`Cash Moves` form loads. Enter the following information on the form:

- :guilabel:`User`: Select the user depositing cash into their account from the drop-down menu. If
  the user is not in the database, they can be created by typing their name in the :guilabel:`User`
  field, and clicking either :guilabel:`Create "user"` or :guilabel:`Create and edit...` to create
  the user and edit the :guilabel:`Create User` form.

> - :guilabel:`Date`: Using the calendar popover, select the date the transaction occurred.

Reviewer: ```suggestion
- :guilabel:`Date`: Select the date the transaction occurred using the calendar popover.
```

Reviewer: to match the other bullets, with action/verb first

Author: We landed on that term i think a year or so ago (and we probalby change dourmind several times again) =D

:menuselection:`Lunch app --> Manager --> Control Accounts.`

All transactions are grouped :guilabel:`By Employee`, and listed alphabetically by the user's first
name. At the end of the user's name, their available balance is displayed. This takes into account

> all deposits to their lunch account, and all transactions within the **Lunch** app.

Reviewer: ```suggestion
name. Their available balance is displayed in the _Amount_ column. This balance takes into account all deposits and transactions to their lunch account.
```

Reviewer: - imo _Amount_ column is more specific, but if "end of the user's name" is more accessibility-friendly stick with that!
- "all transactions within the **Lunch** app": are there transactions outside of the lunch app that are not accounted for?
